### DISCUSSION TITLE
part-ii/2017-67/

### DISCUSSION THREAD

**Discussion by tripos-comments** [tripos-education/maths-tripos-questions#389] [Question comments]
Part II 2017 Galois Theory: Paper 2, Section II, I 

https://questions.tripos.org/part-ii/2017-67/

**Comment by artie2000** [comment]
(a) and (c) are bookwork.

<details>
<summary>(b)</summary>

Take $\alpha=\sqrt{p}+\sqrt{-q}$. Squaring gives $\sqrt{pq}$; then

$\alpha\sqrt{pq}=p\sqrt{q}+q\sqrt{p}$

so we can get $\sqrt{p}$ and $\sqrt{q}$ on their own.

Minimal polynomial is $X^4+2(q-p)X^2+q^2+p^2-pq$.

It's irreducible since $\deg_\mathbb{Q}\alpha=4$ by the tower law.

</details>
